static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-ticker.js

```diff
@@ -1,19 +1,39 @@
 // @flow
 
+import { getLocalCurrencySymbol } from 'lib/geolocation';
+
 export const acquisitionsBannerTickerTemplate = `
-    <div id="banner-ticker" class="js-engagement-banner-ticker engagement-banner-ticker">
-        <div class="js-ticker-so-far engagement-banner-ticker__so-far">
-            <div class="js-ticker-count engagement-banner-ticker__count">$0</div>
-            <div class="engagement-banner-ticker__count-label">contributed</div>
+    <div id="banner-ticker" class="js-engagement-banner-ticker engagement-banner-ticker is-hidden">
+        
+        <div class="js-ticker-under-goal is-hidden">
+            <div class="js-ticker-so-far engagement-banner-ticker__so-far">
+                <div class="js-ticker-count engagement-banner-ticker__count">${getLocalCurrencySymbol()}0</div>
+                <div class="engagement-banner-ticker__count-label">contributed</div>
+            </div>
+            
+            <div class="js-ticker-goal engagement-banner-ticker__goal">
+                <div class="js-ticker-count engagement-banner-ticker__count">${getLocalCurrencySymbol()}0</div>
+                <div class="engagement-banner-ticker__count-label">our goal</div>
+            </div>
         </div>
         
-        <div class="js-ticker-goal engagement-banner-ticker__goal">
-            <div class="js-ticker-count engagement-banner-ticker__count">$0</div>
-            <div class="engagement-banner-ticker__count-label">our goal</div>
+        <div class="js-ticker-over-goal is-hidden">
+            <div class="engagement-banner-ticker__thankyou">
+                <div>Help us beat our goal - thank you</div>
+                <div>Please keep contributing into the new year</div>
+            </div>
+            
+            <div class="js-ticker-exceeded engagement-banner-ticker__exceeded">
+                <div class="js-ticker-count engagement-banner-ticker__count">${getLocalCurrencySymbol()}0</div>
+                <div class="engagement-banner-ticker__count-label">contributed</div>
+            </div>
         </div>
         
-        <div class="engagement-banner-ticker__progress">
-            <div class="js-ticker-filled-progress engagement-banner-ticker__filled-progress"></div>
+        <div class="engagement-banner-ticker__progress-container">
+            <div class="engagement-banner-ticker__progress">
+                <div class="js-ticker-filled-progress filled-progress engagement-banner-ticker__filled-progress ticker__filled-progress-under"></div>
```

Comment:
what's the extra `filled-progress` class for?

Reply:
in the case of the banner, nothing. It's used by the epic to control the colour of the bar. The javascript may replace this with `ticker__filled-progress-over` regardless of whether it's banner or epic so it felt sensible to put it here anyway...

Reply:
hmm I mean the one that's just `filled-progress`, after `js-ticker-filled-progress`

Reply:
oh I see, I have no idea where that came from! Will remove